webextensions/api/extension.json

```diff
@@ -322,7 +322,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/extension/lastError",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "5"
```

Comment:
❌ https://github.com/chromium/chromium/commit/ce95d0324ab1b5c30ff21ff9ef21b073e0f9d030 in [4.0.212.1](https://github.com/chromium/chromium/releases/tag/4.0.212.1) before 4.0.223.

```suggestion
                "version_added": "4"
```